app/code/core/Mage/Adminhtml/Model/Config.php

```diff
@@ -80,16 +89,67 @@ public function getTabs()
         return $this->_tabs;
     }
 
+    /**
+     * @param array $params
+     */
+    public function __construct(array $params = array())
+    {
+        $this->_cacheChecksum = null;
+        $this->setCache(Mage::app()->getCache());
+        $this->setCacheTags([Mage_Core_Model_Config::CACHE_TAG]);
+        $usesCache = Mage::app()->useCache('config');
+        if (!$usesCache || !$this->loadCache()) {
+            /** @var Mage_Core_Model_Config_Base $config */
+            $this->_config = Mage::getConfig()->loadModulesConfiguration('system.xml')
+                ->applyExtends();
+            if ($usesCache) {
+                $this->saveCache();
+            }
+        }
```

Comment:
Missing call to parent constructor

Reply:
We don't need call to Varien_Simplexml_Config as it loads and parses string inside object itself.
We use external config object. That's a trick

Reply:
Feel free to mark this as resolved if you're sure about it.